src/enums.ts

```diff
@@ -0,0 +1,159 @@
+// This file was automatically generated by graphql transpiler
+
+export enum TagFormat {
```

Comment:
@oopsidoodles Are you committing enum.ts ? We decided not to do that, we shall generate the enums.ts and publish it only in the build folder. It's not supposed to be in the SDK. It is generated in real time before every npm publish of the repository.

Reply:
@bholagabbar As it is now, it is generated every publish as it is part of prebuild script. I did remove enums.ts from being committed, although I don't think it would have made much difference as it would change on the next build as soon as backend enums change.